### PR TITLE
Add turtlebot to examples

### PR DESCRIPTION
**Pull request by codemonkey9000**
In this PR:
* Added Turtlebot to Examples page
* Added Turtlebot image
* Added gazebo quay image link
* Fixed homepage Example links (renaming in previous PR broke the anchors)

PR Review Checklist:
- [x] Test search functionality on desktop
- [ ] Test search functionality on tablet
- [ ] Test search functionality on mobile
- [ ] Test filter functionality
- [ ] Test mobile header navigation
- [ ] Test tablet header navigation
- [x] Test desktop header navigation
- [x] Test desktop homepage navigation
- [ ] Run blc http://localhost:4567/ -ro (runs broken-link-checker on whatever port you're using)

### CODE REVIEW

**Comment by codemonkey9000** [comment]
Need to double check the gazebo-turtlebot-sample links once that repo gets published on Tuesday.

**Comment by codemonkey9000** [comment]
Links verified after repo push today.

File: source/includes/examples/_turtlebot.html.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+Gazebo Turtlebot demo on [beta.bons.ai][4].
+
+In this example, we'll walk you through the various Inkling components and
+simulation scripts that are part of a sample implementation of training
+[Turtlebot][2] to navigate the Willow Garage office on the Bonsai Platform.
+Turtlebot is a low-cost mobile base for basic robotics and AI-driven robotics
+R&D. This is a real-world example of how to use the Bonsai Platform for robotics
+using [Gazebo and ROS][3] (Robotic Operating System).
+
+This Example also includes scripts for integrating Gazebo and ROS into the
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: example

File: source/includes/examples/_turtlebot.html.md [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+    Float32 heading_y,
+    Float32 heading_yaw,
+    Float32 yaw,
+    Float32 x,
+    Float32 y
+end
+```
+
+The `State` schema provides the BRAIN with the current x, y coordinates of the
+turtlebot, as well as its orientation. We also supply a heading vector to and
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: Typo around 'to and .."

File: source/includes/examples/_turtlebot.html.md [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+
+```inkling
+concept go_to_point is estimator
+    predicts (Command)
+    follows input(State)
+    feeds output
+end
+```
+
+This concept is named `go_to_point` which predicts a `Command` given a `State`
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: add comma?

File: source/includes/examples/_turtlebot.html.md [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+    Float32 ang_vel
+end
+```
+
+The `Command` schema defines `lin_vel` as the linear velocity and `ang_vel` as
+the angular velocity of Turtlebot in the simulation. These are the two actions
+that can be taken to move Turtlebot. Our BRAIN will output the desired linear
+and angular velocities in m/s and rad/s, respectively. A PID controller will
+make a best effort to achieve these targets within the time limit of the iteration.
+
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: nice

File: source/includes/examples/_turtlebot.html.md [hunk 5]
```diff
@@ -0,0 +1,174 @@
...
+This Python file extends the BonsaiROS interface with methods for controlling
+Gazebo simulations in accordance with the Bonsai simulator protocol.
+
+The purpose of this file is to extend `bonsai_ros.py` to also handle interacting
+with the Gazebo specific services and topics made available when using a Gazebo
+simulation with ROS.
+
+See the [**bonsai_gazebo.py file**][5] on GitHub.
+
+### turtlebot_office_experiment.py
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: turtlebot_office_simulator.py

File: source/includes/examples/_turtlebot.html.md [hunk 6]
```diff
@@ -0,0 +1,174 @@
...
+
+> Source code too long to display, please click Github link.
+
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: This is very subjective, but perhaps starting off with something that emphasizes this is the simulator functionality that drives AI training and prediction.

File: source/includes/examples/_turtlebot.html.md [hunk 7]
```diff
@@ -0,0 +1,174 @@
...
+This is a Python file for integrating the Gazebo simulator through ROS into
+the Bonsai AI Engine. This file repeatedly runs each "episode" and the `State`
+of the simulation in Gazebo gets passed through ROS to the Bonsai AI Engine
+which then proceeds with the next `Command` for Turtlebot and passes that back
+through ROS to Gazebo. 
+
+See the [**turtlebot_office_experiment.py file**][7] on GitHub.
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: turtlebot_office_simulator.py

File: source/includes/examples/_turtlebot.html.md [hunk 8]
```diff
@@ -0,0 +1,174 @@
...
+
+[1]: https://github.com/BonsaiAI/gazebo-turtlebot-sample
+[2]: http://www.turtlebot.com/
+[3]: http://gazebosim.org/tutorials?tut=ros_overview
+[4]: https://beta.bons.ai/new
+[5]: https://github.com/BonsaiAI/gazebo-turtlebot-sample/blob/master/scripts/bonsai_gazebo.py
+[6]: https://github.com/BonsaiAI/gazebo-turtlebot-sample/blob/master/scripts/bonsai_ros.py
+[7]: https://github.com/BonsaiAI/gazebo-turtlebot-sample/blob/master/scripts/turtlebot_office_experiment.py
```
Comment (hodgerpodger) [inline source/includes/examples/_turtlebot.html.md]: turtlebot_office_simulator.py

**Comment by codemonkey9000** [comment]
The problem with updating them to turtlebot_office_simulator is that currently the repo has the previous version, new version won't be published until Tuesday.

But assuming no one is going to use the simulator before Tuesday anyways, I think that's a broken link I'm willing to deal with in order to avoid 2 PRs. :)

**Pull request merged**